sdk/transport_policies/curl/src/az_curl.c

```diff
@@ -121,20 +124,20 @@ az_add_header_to_curl_list(az_pair header, struct curl_slist ** p_list, az_span
     size_t const buffer_size = (size_t)az_span_length(header.key)
         + (size_t)az_span_length(separator) + (size_t)az_span_length(header.value) + 1;
```

Comment:
Why +1 at the end? Add comment or use sizeof(???). Also, if this is CRLF, then it is +2; not +1.